overlord/ifacestate/ifacemgr.go

```diff
@@ -196,6 +209,8 @@ func (m *InterfaceManager) ConnectionStates() (connStateByRef map[string]Connect
 			ByGadget:  cstate.ByGadget,
 			Interface: cstate.Interface,
 			Undesired: cstate.Undesired,
+			PlugAttrs: mergeAttrs(cstate.StaticPlugAttrs, cstate.DynamicPlugAttrs),
+			SlotAttrs: mergeAttrs(cstate.StaticSlotAttrs, cstate.DynamicSlotAttrs),
```

Comment:
I would slightly prefer this merging to happen in daemon. With a note btw that the two sets are expected to be disjoint.